modules/nw-ingress-configuring-application-domain.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+// * installing_aws
+
+[id="nw-ingress-configuring-application-domain_{context}"]
+= Configuring application domain for the Ingress Controller Operator on AWS
+
+//Restricted OpenShift Dedicated or Amazon RH OpenShift cluster administrator
+
+As a cluster administrator, you can specify an alternative default domain for user-created routes by configuring `appsDomain` field. If you specify an alternative domain, it overrides the default cluster domain for the purpose of determining the default host for a newly created route.
+
+For example, you can use the DNS domain for your company as the default domain for routes and ingresses for applications running on your cluster.
+
+.Prerequisites
+
+//* You deployed an {OSD} cluster.
+* You deployed an {product-title} cluster on AWS infrastructure.
+* You installed the `oc` command line interface.
+
+.Procedure
+
+Configure the `appsDomain` field by specifying an alternative default domain for user-created routes.
+
+. Edit the Ingress Controller Operator:
++
+[source,terminal]
+----
+$ oc edit ingresses.config/cluster -o yaml
+----
++
+. Edit the YAML file:
++
+.Sample `appsDomain` configuration to `apps.acme.io`
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Ingress
+metadata:
+  name: cluster
+spec:
+  domain: apps.<domain_url>
+  appsDomain: apps.acme.io
+----
+
+////
+Remove:
+
+. Access the Ingress Controller Operator JSON file:
+[source,terminal]
+----
+$ oc get ingresses.config/cluster -o json | jq .spec
+{
+  "appsDomain": "apps.fidata.io",
+  "domain": "apps.<username>-dd-mm-yy.<url>"
+}
+----
```

Comment:
better to add one note or step here: 
before exposing the route we should wait for the openshift-apiserver finish rolling update.
see https://bugzilla.redhat.com/show_bug.cgi?id=1891952#c5

Reply:
Thank you! I added the note and updated the procedure to keep it consistent with the YAML changes.